src/clickhouse.js

```diff
@@ -311,7 +312,7 @@ ClickHouse.prototype.query = function (chQuery, options, cb) {
 	reqData.format = options.format;
 
 	// use query string to submit ClickHouse query — useful to mock CH server
-	if (this.options.useQueryString) {
+	if (options.readonly) {
```

Comment:
please also check for `useQueryString` for compatibility reasons

Reply:
Fixed